components/prize_pool/wikis/apexlegends/prize_pool_legacy_custom.lua

```diff
@@ -22,13 +22,14 @@ function CustomLegacyPrizePool.customHeader(newArgs, data, header)
 	if Logic.readBool(header.seed) then
 		PrizePoolLegacy.assignType(newArgs, 'seed', 'seed')
 	end
-
+	CLEAN_PLACE = Logic.readBool(header.cleanplace)
 	return newArgs
 end
 
 function CustomLegacyPrizePool.customSlot(newData, CACHED_DATA, slot)
-	newData["place"] = nil
+	if Logic.readBool(CLEAN_PLACE) then
```

Comment:
Logic.readBool here is redundant since you already do it above

Reply:
Noted and corrected